src/db/opt.rs

```diff
@@ -0,0 +1,54 @@
+use super::*;
+
+#[derive(Clone)]
+pub struct AgateOptions {
+    pub path: PathBuf,
+    // TODO: docs
+    pub in_memory: bool,
+    pub sync_writes: bool,
+
+    pub create_if_not_exists: bool,
+    pub num_memtables: usize,
+    pub mem_table_size: u64,
```

Comment:
How about using the same name as badger?

Reply:
I've added all supported options in develop branch. Now they have the same naming convention as badger.